test/AddressRegistry.Api.Legacy.Tests/AddressMatch/AdresMatchSanitizationTests.cs

```diff
@@ -434,30 +608,31 @@ private Task<List<AdresListFilterStub>> MockedSanitizationTest(string straatnaam
 
                     _arrangeStraatnaam(straatnaam);
 
-                    List<AdresListFilterStub> filters = new List<AdresListFilterStub>();
+                    var filters = new List<AdresListFilterStub>();
 
-                    _latest.Arrange(Produce.EmptyList<AddressDetailItem>(),
-                        (when, x) => when.InterceptingLatestAdresQuery((straatnaamid, huisnr, busnr) => filters.Add(new AdresListFilterStub { HuisnummerFilter = huisnr, BusnummerFilter = busnr }), x));
+                    Latest.Arrange(
+                        Produce.EmptyList<AddressDetailItem>(),
+                        (when, x) =>
+                            when.InterceptingLatestAdresQuery(
+                                (straatnaamid, huisnr, busnr) => filters.Add(new AdresListFilterStub
+                                {
+                                    HuisnummerFilter = huisnr,
+                                    BusnummerFilter = busnr
 
-                    AddressMatchRequest request = new AddressMatchRequest
-                    {
-                        Gemeentenaam = existingGemeentenaam,
-                        Straatnaam = straatnaam,
-                        Huisnummer = huisNummer,
-                        Index = index
-                    };
+                                }), x));
 
-                    var notImportant = (OkObjectResult)(Send(request).Result);
+                    // TODO: Why are we doing this?
+                    var notImportant = (OkObjectResult)Send(request).Result;
```

Comment:
Why are we doing this? It's not even an `await Send`

Reply:
I think this has to be done to get the results in the arranged mock.
But you are right, this should be `await` or `.GetAwaiter().GetResult()`